### PR TITLE
Rename `accounts and addresses`

### LINKED ISSUE #1909: Rename `accounts_and_addresses`
## Description

Since `accounts` were removed (at the wallet level) change this directory to a more appropriate name.

## Motivation

Cleanup.

## Are you planning to do it yourself in a pull request?

Yes/No.

### PR DESCRIPTION
**Pull request by Alex6323**
## Links to any relevant issues

Closes #1909

### CODE REVIEW

File: bindings/python/examples/how_tos/wallet/send_amount.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 from iota_sdk import Wallet, WalletOptions, Utils, NodeIndexerAPI, SyncOptions, WalletSyncOptions, SendParams
 
-# In this example we send funds from an account wallet.
```
Comment (Thoralf-M) [inline bindings/python/examples/how_tos/wallet/send_amount.py]: This should stay, it's sending from an output owned by the account output

Comment (Thoralf-M) [reply bindings/python/examples/how_tos/wallet/send_amount.py]: The file should also be in the account_output folder

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Failing CI

File: bindings/python/examples/how_tos/wallet/send_amount.py (hunk 1 above)
Comment (Alex6323) [reply bindings/python/examples/how_tos/wallet/send_amount.py]: Done

**Review by Thoralf-M — APPROVED** [review]

**Review by DaughterOfMars — DISMISSED** [review]

**Review by Thoralf-M — CHANGES_REQUESTED** [review]
Nodejs format

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Nodejs format

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**